homeassistant/components/picnic/strings.json

```diff
@@ -1,7 +1,7 @@
 {
   "config": {
     "step": {
-      "user": {
+      "authenticate": {
```

Comment:
Is this correct or need to be updated back to "user"? I can't tell where this step id is used.

Reply:
You're right indeed, will update it!